app/controllers/api/v4/invoices_controller.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+module Api
+  module V4
+    class InvoicesController < ApplicationController
```

Comment:
Since we're adding the ability to create and load single invoices, we should also add the ability to list invoices. If we implement it similarly to Stripe cards, we can have a route for indexing by organization as well as by user:
https://github.com/hackclub/hcb/blob/70685d707103fe83fde631f9b5fc3a1803a2b271/app/controllers/api/v4/stripe_cards_controller.rb#L6-L15